### PR TITLE
Dreamchecker and Various Fixes

### PR DESCRIPTION
**Pull request by Geevies**
* Implements Dreamchecker which can check your code without needing to send it to Drone each time.
* Fixes various bugs found by Dreamchecker, do double check this code.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/mob/holder.dm [hunk 1]
```diff
@@ -239,8 +239,6 @@ var/list/holder_mob_icon_cache = list()
 
 		post_scoop()
 
-		return success
-
```
Comment (Arrow768) [inline code/modules/mob/holder.dm:243]: I am unsure why that return has been removed.
What error/message did you get there?

Comment (123khadi) [reply code/modules/mob/holder.dm:243]: Graater and small eror will be occur

**Pull request closed without merging**